### PR TITLE
Fixes #6161 - Mark override on adding smart variable override via api

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

File: app/controllers/api/v2/override_values_controller.rb [hunk 1]
```diff
@@ -43,6 +43,7 @@ def show
 
       def create
         @override_value = @smart.lookup_values.create!(params[:override_value])
+        @smart.override = true
```
Comment (isratrade) [inline app/controllers/api/v2/override_values_controller.rb]: @tbrisker, I don't think this line will persist anything in the database. I think you may want this instead.

``` ruby
@smart.update_attributes(:override => true)
```

Comment (tbrisker) [inline app/controllers/api/v2/override_values_controller.rb]: Thanks, corrected.

**Comment by isratrade** [comment]
Merged as 73eb580f9e910736ec797b17ee0cead0dcb12856, thanks @tbrisker !

**Comment by domcleal** [comment]
Please don't forget tests in the future, thanks.

**Comment by isratrade** [comment]
@tbrisker, can you add a test for this and submit as a separate PR.

**Pull request closed without merging**